prql-python/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+use prql_compiler::compile;
+use pyo3::exceptions;
+use pyo3::prelude::*;
+
+#[pyfunction]
+pub fn to_sql(query: &str) -> PyResult<String> {
+    match compile(query) {
+        Ok(sql) => Ok(sql.replace('\n', " ").replace('\t', " ")),
+        Err(err) => Err(PyErr::new::<exceptions::PySyntaxError, _>(format!(
+            "{}",
+            err
+        ))),
+    }
+}
+
+#[pymodule]
+fn prql_python(_py: Python, m: &PyModule) -> PyResult<()> {
+    m.add_function(wrap_pyfunction!(to_sql, m)?)?;
+    Ok(())
+}
+// This test below is causing a linking error here https://github.com/qorrect/prql/runs/6099160429?check_suite_focus=true
```

Comment:
Great for putting in the PR, let's merge and iterate. Maybe others can take a glance, I'm happy to take a look in a couple days otherwise.